apps/web/pages/_document.tsx

```diff
@@ -3,18 +3,13 @@ import Document, { DocumentContext, Head, Html, Main, NextScript, DocumentProps
 type Props = Record<string, unknown> & DocumentProps;
 
 function toRunBeforeReactOnClient() {
-  const calEmbedMode = location.search.includes("embed=");
-  try {
-    // eslint-disable-next-line @calcom/eslint/avoid-web-storage
-    window.sessionStorage.setItem("calEmbedMode", String(calEmbedMode));
-  } catch (e) {}
+  const calEmbedMode =
+    location.search.includes("embed=") ||
+    /* Iframe Name */
+    window.name.includes("cal-embed");
```

Comment:
Use more reliable approach of embed detection that works in Chrome Incognito as well.
As chrome incognito doesn't have sessionStorage available, this is the best approach.

window.name is the name attribute of the iframe. So, it is guaranteed to be cal-embed